ligo/haskell/test/README.md

```diff
@@ -1,4 +1,4 @@
-# BaseDAO LIGO tests
+# BaseDAO LIGO meta
 
 BaseDAO LIGO tests are implemented in Haskell, using `cleveland` [library](https://gitlab.com/morley-framework/morley/-/tree/master/code/cleveland).
```

Comment:
This probably should have not be renamed and left as `BaseDAO LIGO tests` seeing as this README is specifically under `haskell/tests` and the "Testing" section in `ligo/README` points here.

Reply:
Oh right, good catch.